defaults/main.yml

```diff
@@ -25,6 +25,12 @@ alpaca_karaf_deploy_dir: "{{ alpaca_karaf_dir }}/deploy"
 alpaca_karaf_user: karaf
 alpaca_local_mvn_path: /opt/maven/repo
 
+alpaca_triplestore_base_url: http://localhost:8080/bigdata
```

Comment:
Probably should be consistent on whether to end with a slash or not. Unless we can adjust to account for either